docs/apache-airflow/core-concepts/params.rst

```diff
@@ -328,6 +329,17 @@ For examples also please take a look to two example DAGs provided: ``example_par
 The trigger form can also be forced to be displayed also if no params are defined using the configuration switch
 ``webserver.show_trigger_form_if_no_params``.
 
+.. versionadded:: 2.8.0
```

Comment:
I think this should be `versionchanged`? And the block below should be intended.

Reply:
Oh, this smells like a general follow-up - I see this in various manners being used for "versionchanged" and "versionadded" that these blocks are used as simple markers and indented. Shall we have a PR which aligns this (outside of this PR) across the repo?